wiki/webserver/ExternalWebserversFile.md

```diff
@@ -55,19 +58,21 @@ server {
   # path to bluemap-webroot, bluemap can also be used in a sub-folder .. just adapt the paths accordingly
   root /var/www;
   
-  location / {
-    try_files $uri $uri/ =404 ;
+  location ~* ^/maps/[^/]*/tiles/ {
+    # High-res tiles are stored as precompressed JSON with a fallback to returning an empty tile.
+    # Low-res tiles are stored as pngs with a fallback to returning 204 (No Content).
+    location ~* \.json$  {
+      error_page 404 =200 /assets/emptyTile.json;
+      gzip_static always;
+    }
+    location ~* \.png$ {
+      try_files $uri =204;
+    }
   }
-  
-  # map-tiles are stored compressed, and they have a fallback file that should be returned if the tile does not exist
-  location /maps/ {
-    error_page 404 =200 /assets/emptyTile.json;
-    gzip_static always;
-  }
-  
-  # Proxy requests to the live data interface of each map to bluemaps integrated webserver
-  # If you have multiple servers you will need to proxy each map-id to the correct server
-  location ~* /(maps/[^/]*/live/.*) {
+
+  # Proxy requests for live data to the bluemaps integrated webserver.
+  # If you have multiple servers you will need to proxy each map-id to the correct server.
+  location ~* ^/maps/[^/]*/live/ {
     proxy_pass http://127.0.0.1:8100/$1;
   }
```

Comment:
Does this still work? Iirc, the `$1` in the proxypass-url references the first group from the regex, but you removed the group? ^^

Reply:
Good catch, I missed that line when copy/pasting into the doc :)